README.md

```diff
@@ -3,5 +3,7 @@ In this assignment, you'll design and implement one of the string manipulation f
 Remember that a string is an array of characters. Algorithms that worked on restricted arrays will work on strings as well.
 
 ## Exercise
-Design and implement a method to reverse a string *in place*. For example, if the method is called with input parameter of "Lovelace", the method should update the input string object to have the value "ecalevoL". Share and explain the time and space complexities for your solution.
-**Note**: Do not use Ruby provided functionality for `.reverse` and `.reverse!`. You may use `.length`. 
+* Design and implement a method to reverse a string *in place*. For example, if the method is called with input parameter of "Lovelace", the method should update the input string object to have the value "ecalevoL".
+* Share and explain the time and space complexities for your solution.
+* At minimum, your implementation should pass the basic tests.
+**Note**: Do not use Ruby provided functionality for `.reverse` and `.reverse!`. You may use `.length`.
```

Comment:
Similar comment to one of @kariabancroft's comments on the `reverse_words` repo: It seems likely that we want an extra blank line above this so that the note ends up in its own paragraph.